en/concept/system_startup.md

```diff
@@ -68,33 +68,45 @@ The resulting boot log has detailed information about the boot sequence and shou
 
 #### Common boot failure causes
 
-  * For custom applications: The system was out of RAM. Run the `free` command to see the amount of free RAM.
-  * A software fault or assertion resulting in a stack trace
+- For custom applications: The system was out of RAM. Run the `free` command to see the amount of free RAM.
+- A software fault or assertion resulting in a stack trace
 
 ### Replacing the System Startup
 
-In most cases customizing the default boot is the better approach, which is documented below. If the complete boot should be replaced, create a file `/fs/microsd/etc/rc.txt`, which is located in the `etc` folder on the microSD card. If this file is present nothing in the system will be auto-started.
+In most cases customizing the default boot is the better approach, which is documented below.
+If the complete boot should be replaced, create a file `/fs/microsd/etc/rc.txt`, which is located in the `etc` folder on the microSD card. If this file is present nothing in the system will be auto-started.
 
 ### Customizing the System Startup
 
 The best way to customize the system startup is to introduce a [new airframe configuration](../dev_airframes/adding_a_new_frame.md).
-If only tweaks are wanted (like starting one more application or just using a different mixer) special hooks in the startup can be used.
+If only tweaks are wanted (like starting one more application) special hooks in the startup can be used.
 
 :::warning
-The system boot files are UNIX FILES which require UNIX LINE ENDINGS. If editing on Windows use a suitable editor.
+The system boot files are UNIX FILES which require UNIX LINE ENDINGS.
+If editing on Windows use a suitable editor.
 :::
 
-There are three main hooks. Note that the root folder of the microsd card is identified by the path `/fs/microsd`.
+There are two main hooks.
+Note that the root folder of the microsd card is identified by the path `/fs/microsd`.
 
-* /fs/microsd/etc/config.txt
-* /fs/microsd/etc/extras.txt
-* /fs/microsd/etc/mixers/NAME_OF_MIXER
+- /fs/microsd/etc/config.txt
+- /fs/microsd/etc/extras.txt
 
 #### Customizing the Configuration (config.txt)
 
-The `config.txt` file can be used to modify shell variables.
+The `config.txt` file can be used to modify shell variables (parameters).
```

Comment:
Maybe shift this to modifying parameters? There are a few things left that could be set with shell variables, but for the most part it's all moving to parameters.

Reply:
@dagar Absolutely. I was reading this as "using the shell to modify parameters".

```suggestion
The `config.txt` file can be used to modify parameters.
```